source/Jobbr.Storage.MsSql.Tests/ExposeStorageProvider.cs

```diff
@@ -8,14 +8,14 @@ public partial class ServerRegistrationTests
         public class ExposeStorageProvider : IJobbrComponent
         {
             internal readonly IJobStorageProvider JobStorageProvider;

```

Comment:
Naming convention for fields would be `_jobStorageProvider`

Reply:
I kept it because it's internal and it's actually accessed like this ExposeStorageProvider.Instance.JobStorageProvider

Reply:
Would it then make sense to convert it to a property instead?